change/@azure-msal-browser-2ca47cd6-90a9-4988-b2c3-d0fbeacecdad.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
I would consider this more of an enhancement, as opposed to a new feature.

Reply:
Changed.